tools/doc/generate.js

```diff
@@ -88,6 +88,13 @@ function next(er, input) {
       );
       break;
 
+    case 'man':
+      require('./man.js')(input, inputFile, function(er, manSrc) {
+        console.log(manSrc);
+        if (er) throw er;
```

Comment:
@mscdex lol, you'd think :D

I was looking at `case 'json':` when I wrote that, trying to mirror surrounding code style; perhaps that one's wrong too?